src/components/Modals/components/Modal.js

```diff
@@ -32,12 +32,11 @@ export const Modal = ({
   const okButton = useRef(null);
 
   useEffect(() => {
-    okButton.current?.focus();
+    const button = okButton.current;
```

Comment:
I think the variable name is much clearer now :thumbsup:

I probably wasn't as clear as I should have been in my original comment in #29. Basically I was just suggesting making the comment more descriptive so that it's clearer why the `ref` needs to be copied to a variable before being used in the clean-up function. I personally wasn't able to understand much from "ref must be saved to variable to accomodate react warning" because I didn't know what warning the comment was referring to.

I see that you removed the comment now, but maybe it would still be useful to have one since it's kind of a particularity of React that (as far as I can tell) isn't really covered in the official React docs.

Reply:
Sorry, didn't catch that. Due to the `autoFocus` attribute we do not need the useRef() anymore so now everything is gone. I have also struggled a lot to find any documentation on this. Only the issue I found and linked to the #29 PR. I paste it here as well for future reference: https://github.com/facebook/react/issues/15841